package.json

```diff
@@ -21,7 +21,8 @@
     "clsx": "^2.0.0",
     "prism-react-renderer": "^2.3.0",
     "react": "^18.0.0",
-    "react-dom": "^18.0.0"
+    "react-dom": "^18.0.0",
+    "sermas-eu-github-io": "file:"
```

Comment:
Is this necessary?

Reply:
line 24 should not be necessary, for line 25 it si being added automatically by the package manager.
The both of these modifications have been added automatically by the framework.

Reply:
ok thank you, I was not aware of it